moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -717,13 +717,6 @@ const ProjectFundingTable = () => {
             updateProjectFundingData.funding_amount = Number(
               newData.funding_amount
             );
-            // add fallback of empty strings instead of null value
-            updateProjectFundingData.fund_dept_unit =
-              newData.fund_dept_unit || "";
-            updateProjectFundingData.funding_description =
-              newData.funding_description || "";
-            updateProjectFundingData.funding_program_id =
-              newData.funding_program_id || 0;
```

Comment:
looks like we're still storing empty strings and zeroes in the DB when editing a funding item. i think what these lines of code actually need to do is cast empty strings to `null`, e.g.

```javascript
updateProjectFundingData.fund_dept_unit = newData.fund_dept_unit || null;
```

Reply:
i can open a separate issue to fix the funding data tech debt if you prefer

Reply:
the only one that's giving me trouble is the `amount` column. if i edit that from a number to blank, it bombs hasura b/c of an empty string:

<img width="715" alt="Screen Shot 2023-01-18 at 11 09 11 AM" src="https://user-images.githubusercontent.com/14793120/213229927-ebb1c3ff-5158-4f7c-bf90-4c3aaefcd64a.png">

Reply:
☝️ this can also occur when creating a new row, if you set the amount field then clear it out before saving the row